src/Log/SentryLogger.php

```diff
@@ -332,4 +333,80 @@ public static function backtrace(array $record) : array
         ]);
     }
 
+    /**
+     * Get the information about the current release, if possible
+     * @return array
```

Comment:
I'm using casted returns everywhere these days, so an appended `: array` would be grand :-)